package.json

```diff
@@ -82,7 +82,6 @@
     "sade": "^1.8.1",
     "tailwindcss": "^3.4.18",
     "tinyspy": "^4.0.4",
-    "tsx": "^4.20.6",
```

Comment:
@ARYPROGRAMMER can you run `pnpm i` so the `pnpm-lock.yaml` gets updated as well? Include that in the PR.